addons/shortcuts/statics.lua

```diff
@@ -198,9 +198,10 @@ end
 -- Iterate through resources and make validabils.
 function validabils_it(resource)
     for id,v in pairs(res[resource]) do
-        if (not v.monster_level and v.prefix) or (v.monster_level and v.monster_level ~= -1 and v.ja:sub(1,1) ~= '#' ) then
+        if (not v.monster_level and v.prefix) or (v.monster_level and v.monster_level ~= -1 and v.ja:sub(1,1) ~= '#' ) or (v.category == 'Usable') then
         -- Monster Abilities contains a large number of player-usable moves (but not monstrosity-usable). This excludes them.
-            make_abil(strip(v.english),resource,id)
+            local name = resource ~= 'items' and strip_non_alphanumeric_convert_digits_to_roman(v.english) or strip_non_alphanumeric_keep_plus(v.english)
```

Comment:
Are there name collisions if you don't keep the plus and avoid converting digits to roman numerals?

Reply:
I didn't try, but I felt `/rolandaifuku+1` felt more natural than `/rolandaifuku1`.

Reply:
That's fine, but you can use `/rolandaifuku+1` either way.

Reply:
No. If we only used the original, it would be simply /rolandaifuku for both the normal and +1 versions: the plus gets stripped and the 1 gets omitted when converting to Roman numerals.